### DISCUSSION TITLE
How to extract sound effects from an audio track?

### DISCUSSION THREAD

**Discussion by jordanttt** [Anjok07/ultimatevocalremovergui#1028] [Q&A]
Hey guys.

I was curious if there are any models to extract sound effects from an audio track. I don't want any voices, music or whatever. Just the sound effects.

So let's say you would use the GTA VI trailer, then I would want to have the sounds of the vehicles, etc. Not the music in the back or the voice lines.

Anyone got an idea?

**Comment by HopeAndFaithStudios** [comment]
I have the exact same question! I wish there was an easy way to do this. DX I really do love UVR though. I just wish there was an option to keep sound effects, but remove everything else. I also wish I could isolate vocals much better from TV shows (which I cannot find 5.1 surround sound prints of, unfortunately).

**Reply by OnivalSouza** [reply]
Look for a model that separates the stems and test it.

**Reply by HopeAndFaithStudios** [reply]
Which one would you recommend? To be totally honest, I'm still new to this.
